ocaml/xapi/storage_migrate.ml

```diff
@@ -727,8 +727,8 @@ let reqs_outstanding_timeout = 150.0 (* Tapdisk should time out after 2 mins. We
 let pre_deactivate_hook ~dbg ~dp ~sr ~vdi =
   let open State.Send_state in
   let id = State.mirror_id_of (sr,vdi) in
-  let start_time = Oclock.gettime Oclock.monotonic in
-  let get_delta () = (Int64.to_float (Int64.sub (Oclock.gettime Oclock.monotonic) start_time)) /. 1.0e9 in
+  let start_time = Mtime_clock.now () in
+  let get_delta () = Mtime.(span start_time (Mtime_clock.now ()) |> Span.to_s) in
```

Comment:
This works, but looks like [Mtime.count](http://erratique.ch/software/mtime/doc/Mtime_clock.html#VALcount) was designed  exactly for this purpose (returning the time span from a fixed point in time), i.e.
```ocaml
let start = Mtime.counter () in
let get_delta () = Mtime.count start in
```